src/main/resources/assets/iris/lang/zh_tw.json

```diff
@@ -20,21 +20,21 @@
   "options.iris.refreshShaderPacks": "重新整理",
   "options.iris.shaderPackSelection": "光影包……",
   "options.iris.shaderPackSelection.title": "光影包",
-  "options.iris.shaderPackSelection.addedPack": "已新增光影包 \"%s\"",
+  "options.iris.shaderPackSelection.addedPack": "已新增「\"%s\"」光影包",
```

Comment:
Isn't it a little redundant to have both types of quotes here?

Reply:
Frankly, maybe.
Since we don't use the type used in English, I think it will be more appropriate to add the Chinese type.
If it might cause confusion or something, `「` and `」` can be replaced by space.

Reply:
Would it then be natural to remove the English type quotes and only leave the Chinese type quotes?

Reply:
Unfortunately, it doesn't apply the translations file if doing so.

Reply:
Nope.

Reply:
> Did you remove the backslashes too when you removed the quotes? Otherwise the escaping breaks.

I didn't remove them. I left the backslashes.

Reply:
You'd have to remove the backslashes. If there are backslashes but no quotes then the file is invalid.

Reply:
> You'd have to remove the backslashes. If there are backslashes but no quotes then the file is invalid.

Just tried it with 1.2.0-rc1. The translations file didn't work properly.